modules/kmm-security.adoc

```diff
@@ -32,7 +32,8 @@ The authorization model for that workload depends on the namespace of the `Modul
 When setting up RBAC permissions, remember that any user or `ServiceAccount` creating a `Module` resource in the `openshift-kmm` namespace results in KMM automatically running privileged workloads on potentially all nodes in the cluster.
 ====
 
-To allow any `ServiceAccount` to use the `privileged` SCC and therefore to run module loader or device plugin pods, use the following command:
+To allow any `ServiceAccount` to use the `privileged` SCC and hence to run worker and / or device plugin Pods, use the
+following command:
```

Comment:
This is another instance where this one line makes this module more of a procedure module. I know you didn't writer or even intentionally touch this line, but after fixing hard-wrapping, I'd suggest reframing this to be less of an instruction:
```suggestion
To allow any `ServiceAccount` to use the `privileged` SCC and run worker or device plugin Pods, you can use the `oc adm policy` command, as in the following example:
```

Reply:
Changed.